rocky/OOI_database_seed.json

```diff
@@ -682,7 +683,6 @@
       "last_updated": "2021-09-29T16:46:09.835Z",
       "data": {
         "description": "This software should not be exposed to the Internet.",
```

Comment:
Ports might carry multiple services, mentioning software here does not cover all usecases. We also in this case dont see the software, we see just the port.